rules/app_competition_rules.tex

```diff
@@ -60,7 +60,7 @@ \subsection{Champions Cup and Challenge Shield}
 In addition, some rules are simplified for Challenge Shield matches. The major differences are:
 \begin{itemize}
   \item The requirement to recognize visual referee signals is relaxed for the Challenge Shield (\cf~\cref{sec:robot_players}).
-  \item Goals may be scored via direct kick from kick-off. The indirect kick rules remain unchanged for the Challenge Shield (\cf~\cref{sec:indirect_kick_challenger}), but were tightened for the Champions Cup.
+  \item Goals may be scored via direct kick from kick-off. The indirect kick rules remain unchanged for the Challenge Shield (\cf~\cref{sec:indirect_kick_old}), but were tightened for the Champions Cup.
```

Comment:
This need to be updated

Reply:
I'm going to push a proposal for an updated rule, but maybe we can also consider dropping this entirely